external-secrets-operator.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: external-secrets-operator
+  version: 0.8.1
+  epoch: 0
+  description: Integrate external secret management systems with Kubernetes
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - git
+      - bash
+      - go
+      - yq
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/external-secrets/external-secrets
+      tag: v${{package.version}}
+      expected-commit: a0d2aef2e35c259c9ee75d65f7587e6ed71ef2ad
+
+  - runs: |
+      # Our global LDFLAGS conflict with a Makefile parameter
+      unset LDFLAGS
```

Comment:
we needed to add this for a previous package but it's not always needed.  Just wanted to check that unsetting the global LDFLAGS was needed for external secrets?

Reply:
I don't think we need to reset the LDFLAGS here ,